sdk/metrics/src/jmh/java/io/opentelemetry/sdk/metrics/MetricsTestOperationBuilder.java

```diff
@@ -23,6 +22,7 @@
  * return a wrapper for both bound and unbound versions of that operation which can then be used in
  * a benchmark.
  */
+@SuppressWarnings("ImmutableEnumChecker")
```

Comment:
Why not adding the `javax.annotation` immutable annotation?

Reply:
Error prone doesn't respect that. And this is only a test so suppressing seems fine.

Reply:
Ok